pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -434,7 +436,8 @@ contract ManagedPool is BaseWeightedPool, AumProtocolFeeCache, ReentrancyGuard {
 
     /**
      * @dev Can enable/disable the LP allowlist. Note that any addresses added to the allowlist
-     * will be retained if the allowlist is toggled off and back on again.
+     * will be retained if the allowlist is toggled off and back on again, because removing addresses
+     * is not allowed while the allowlist is disabled.
```

Comment:
```suggestion
     * will be retained if the allowlist is toggled off and back on again, because adding or removing addresses
     * is not allowed while the allowlist is disabled.
```